app/eventyay/control/urls.py

```diff
@@ -607,6 +607,7 @@
                 url(r'^$', admin.AdminDashboard.as_view(), name='admin.dashboard'),
                 url(r'^organizers/$', admin.OrganizerList.as_view(), name='admin.organizers'),
                 url(r'^events/$', admin.AdminEventList.as_view(), name='admin.events'),
+                url(r'^attendees/$', admin.AttendeeListView.as_view(), name='admin.attendees'),
```

Comment:
Use `path`, not `url`, for new code.

Old code with `url` will be updated somedays.